### PR TITLE
Misc typo fixes

### PR DESCRIPTION
**Pull request by BrettDong**
#### Summary
None

#### Purpose of change
Discovered some obvious misspellings when doing a manual audit on the auxiliary spell checker dictionary.

### CODE REVIEW

File: data/json/items/ammo/223.json [hunk 1]
```diff
@@ -62,7 +62,7 @@
     "copy-from": "223",
     "type": "AMMO",
     "name": { "str_sp": "5.56 NATO M855A1" },
-    "description": "An updated loading of M855 5.56x45mm ammunition, featuring a 62 grain proectile containing an exposed steel penetrator and copper slug.  Designed as a mostly lead-free projectile, the opportunity was further taken to improve upon M855's penetration and lethality chracteristics through penetrator design and increased pressure.  Its improved accuracy, penetration and lethality over M855 are appreciated by users of shorter barreled AR-variants.",
+    "description": "An updated loading of M855 5.56x45mm ammunition, featuring a 62 grain projectile containing an exposed steel penetrator and copper slug.  Designed as a mostly lead-free projectile, the opportunity was further taken to improve upon M855's penetration and lethality chracteristics through penetrator design and increased pressure.  Its improved accuracy, penetration and lethality over M855 are appreciated by users of shorter barreled AR-variants.",
```
Comment (BrettDong) [inline data/json/items/ammo/223.json:65]:
```suggestion
    "description": "An updated loading of M855 5.56x45mm ammunition, featuring a 62 grain projectile containing an exposed steel penetrator and copper slug.  Designed as a mostly lead-free projectile, the opportunity was further taken to improve upon M855's penetration and lethality characteristics through penetrator design and increased pressure.  Its improved accuracy, penetration and lethality over M855 are appreciated by users of shorter barreled AR-variants.",
```

File: data/json/npcs/refugee_center/surface_refugees/NPC_Boris_Borichenko.json [hunk 2]
```diff
@@ -376,7 +376,7 @@
...
+      "[Boris boots up the laptop and opens a folder labelled 'STORY 1'.  In it are numerous text files containing smatterings of stream-of-conscious notes, copy-and-pasted bits of fanfictions, notes about how to write a novel, and more.  The story itself is a handful of poorly fleshed out chapters written with the quality level you'd expect from a grade schooler.  Nonetheless it's quite impressive that there's this much stuff on a single topic.  Boris talks for a while about dinner conversations for weeks that wuold focus on Ash's ideas for his stories.  After a while, he trails off and just continues flipping through the files, reading silently.]",
```
Comment (BrettDong) [inline data/json/npcs/refugee_center/surface_refugees/NPC_Boris_Borichenko.json:379]:
```suggestion
      "[Boris boots up the laptop and opens a folder labelled 'STORY 1'.  In it are numerous text files containing smatterings of stream-of-conscious notes, copy-and-pasted bits of fanfictions, notes about how to write a novel, and more.  The story itself is a handful of poorly fleshed out chapters written with the quality level you'd expect from a grade schooler.  Nonetheless it's quite impressive that there's this much stuff on a single topic.  Boris talks for a while about dinner conversations for weeks that would focus on Ash's ideas for his stories.  After a while, he trails off and just continues flipping through the files, reading silently.]",
```

File: data/json/snippets/graffiti.json [hunk 3]
```diff
@@ -59,7 +59,7 @@
     "category": "shelter_graffiti_snippets",
     "text": [
       { "id": "shelter_graffiti_1", "text": "BIGGEST WASTE OF TAX MONEY FUCK YOU GOVERMINT" },
-      { "id": "shelter_graffiti_2", "text": "Dont eat the proten bars" },
```
Comment (LyleSY) [inline data/json/snippets/graffiti.json:62]: This is fine, graffiti can have bad spelling

File: data/mods/Dark-Skies-Above/snippets/graffiti.json [hunk 4]
```diff
@@ -59,7 +59,7 @@
     "category": "dks_shelter_graffiti_snippets",
     "text": [
       { "id": "dks_shelter_graffiti_1", "text": "BIGGEST WASTE OF TAX MONEY FUCK YOU GOVERMINT" },
-      { "id": "dks_shelter_graffiti_2", "text": "Dont eat the proten bars" },
+      { "id": "dks_shelter_graffiti_2", "text": "Dont eat the protein bars" },
       { "id": "dks_shelter_graffiti_3", "text": "FEMA: FUCKIN EAT MY ASSHOLE" },
```
Comment (LyleSY) [inline data/mods/Dark-Skies-Above/snippets/graffiti.json:63]: Same, graffiti can have bad spelling

**Review by LyleSY — COMMENTED** [review]
I'd leave out the graffiti but otherwise good stuff here

**Review by LyleSY — APPROVED** [review]

**Pull request merged**